### DISCUSSION TITLE
PCBs available

### DISCUSSION THREAD

**Discussion by neohusky** [Dilbert66/esphome-dsckeybus#29] [General]
Hi All

I'm not sure if anyone is interested, but, I've got 3 spare pcbs (without components) that I had made for my project that are leftover.  I used the schematics from [here](https://github.com/taligentx/dscKeybusInterface/tree/master/extras/PCB%20Layouts/WemosD1Mini).  I live in Sydney, Australia and I'm happy to give them to away you live local or ship them at your expense.  

Please reach out. I've attached pics of the PCB and what the finished product looked like.

<img width="743" alt="pcb1" src="https://user-images.githubusercontent.com/1212618/119828169-436c1880-bf3d-11eb-9d29-464e3ece7bc1.png">
<img width="793" alt="pcb2" src="https://user-images.githubusercontent.com/1212618/119828248-541c8e80-bf3d-11eb-8d23-28f733274899.png">
<img width="783" alt="pcb3" src="https://user-images.githubusercontent.com/1212618/119828254-567ee880-bf3d-11eb-9e7d-94a2f85e0838.png">
<img width="596" alt="pcb4" src="https://user-images.githubusercontent.com/1212618/119828258-58e14280-bf3d-11eb-82bd-31a5e462edec.png">
<img width="714" alt="pcb5" src="https://user-images.githubusercontent.com/1212618/119828265-5aab0600-bf3d-11eb-9d7e-b73fa6742b61.png">

**Comment by hndrk** [comment]
Nice work! Still available?
